WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdView.swift

```diff
@@ -118,6 +115,7 @@ private extension BlazeEditAdView {
             )
 
             Text(viewModel.taglineFooterText)
+                .foregroundStyle(viewModel.taglineRemainingLength >= 0 ? .secondary : Color(.error))
```

Comment:
Nice work using the `error` color. 

Should we use `.error` color also when the tagline is empty? Applies to the description field as well. 

<img src="https://github.com/woocommerce/woocommerce-ios/assets/524475/3538a818-9e07-4f6a-8074-f8b36f98aa5c" width="320"/>

Reply:
Good point - I added properties for validating the fields and updated the view to use them in 34aaf628f909d6b2561f8adcb6c78e7792b867b1. Test added in 053d536e370b8bd1184126a70b7407f7298beb1b.